analysis/monk/src/DampenHarm.tsx

```diff
@@ -0,0 +1,73 @@
+import { formatNumber, formatThousands } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+class DampenHarm extends Analyzer {
+  hitsReduced = 0;
+  damageReduced = 0;
+  skip = 0;
+  currentMaxHP = 0;
+
+  constructor(options: Options) {
+    super(options);
+    if (!this.selectedCombatant.hasTalent(SPELLS.DAMPEN_HARM_TALENT.id)) {
+      this.active = false;
+      return;
+    }
+    this.addEventListener(Events.damage.to(SELECTED_PLAYER), this.updateMaxHP);
```

Comment:
If we want higher coverage of this we should add `Events.heal.to` to the list
Up to you if you want to add this or not (it should make it more accurate*)

Reply:
Done